raiden/routing.py

```diff
@@ -38,11 +39,16 @@ def get_best_routes(
     previous_address: Optional[Address],
     pfs_config: Optional[PFSConfig],
     privkey: PrivateKey,
+    our_address_metadata: AddressMetadata,
 ) -> Tuple[Optional[str], List[RouteState], Optional[UUID]]:
 
     token_network = views.get_token_network_by_address(chain_state, token_network_address)
     assert token_network, "The token network must be validated and exist."
 
+    if pfs_config is None or one_to_n_address is None:
```

Comment:
This ain't work just like this and is responsible for the errors in the CI. The tests failing include direct payments which did not need (and dont have) PFS config set. Now direct payments need to communicate to the PFS and thus need PFS config to be set.

Reply:
Thinking about it, I come more and more to the conclusion that we should not require having a PFS config in place for direct payments. 
Same as we can accept routes with missing or incomplete metadata we should be able to send direct payments with wrong or missing metadata. 
Thus the above condition seems to me, should be at the place where it was before

Reply:
because we need a route for mediated payments which is essential for a mediated payment and in the case of direct payments we only need metadata which is not essential to a successful payment (we already know the route)